### DISCUSSION TITLE
Customize blog url when blog enabled

### DISCUSSION THREAD

**Discussion by ctuxboy** [BoidCMS/BoidCMS#41] [General]
When the blog option is enabled, is it possible add an option for add or remove the **blog** or **articles** in the url? Or planned in a next update?
I think it is better for readability(?) i think, so the pages are in url's root, and the blogpost in a sub-url?

Example:
website.com/blog/here-a-blogpost
website.com/articles/here-a-blogpost
website.com/here-a-page

**Comment by sysadev** [comment]
I believe that is left for the admin to consider as there is a specific field namely `permalink` that could be set to any desired slug and the page could only be accessed through it.
For example, when you have a post with the slug `new-post` and you want the blog post to have `blog/` prefix, then you can easily just put `blog/new-post` in the `permalink` field and it will work just fine.

But in the case you might want something auto then a plugin should be the best idea.

**Reply by ctuxboy** [reply]
Wow! okay, i didn't know it.
Stupid that I didn't test this!
Thanks a lot for the help!
